agreement/player.go

```diff
@@ -281,13 +281,16 @@ func (p *player) handleCheckpointEvent(r routerHandle, e checkpointEvent) []acti
 // It returns the time of the lowest credential's arrival, if one was
 // collected and added to lowestCredentialArrivals, or zero otherwise.
 func (p *player) updateCredentialArrivalHistory(r routerHandle, ver protocol.ConsensusVersion) time.Duration {
-	// only append to lowestCredentialArrivals if this was a successful round completing in period 0.
-	if p.Period != 0 {
+	if p.Round < credentialRoundLag {
```

Comment:
if `p.Round` == `credentialRoundLag` then the `credHistoryRound` is 0 that does not make any sense

Reply:
I added an equality test